src/raygun.js

```diff
@@ -56,7 +56,7 @@ var raygunFactory = function(window, $, undefined) {
     _groupingKeyCallback,
     _beforeXHRCallback,
     _afterSendCallback,
-    _raygunApiUrl = 'https://api.raygun.io',
```

Comment:
Could this be a breaking change?

Reply:
Perhaps it something to check with Denny but I believe for the API supports both api.raygun.io (legacy) and api.raygun.com